### ISSUE TITLE
Several bx commands not working

### ISSUE THREAD

**Issue by lucaskuperman** [IBM-Cloud/ibm-cloud-cli-sdk#58]
Hello. Have a T460 w/Win10 and installed bluemix CLI version 0.5.6+5b886b4-2017-08-14T08:16:22+00:00.
Today I needed to run 2 commands and both failed:

1) bx cf ssh 
when trying to ssh a bluemix app, the terminal hangs and gets blocked: keystrokes are no longer recognized, including Ctrl-X and Ctrl-Z or any other combination. I have no other option than to close the unusable terminal.

2) bx service rename
```
C:\Users\LUCASKuperman>bx service rename "dashDB for Analytics-s5" "glaim"
Invoking 'cf rename-service dashDB for Analytics-s5 glaim'...

Renaming service dashDB for Analytics-s5 to glaim in org GLAIM - Global Logistics Americas Information Management / space test as kuperman@ar.ibm.com...
FAILED
Server error, status code: 404, error code: 340001, message: The service instance could not be found: 23c82ba3-ac7d-4958-abe9-0b1cd4de582b
```
I had to ask bluemix admin to rename it for me, using the cf CLI instead

I downloaded installer from bluemix dedicated 'get started' official page (https://console.w3ibm.bluemix.net/apps/f3518642-eec8-44de-a581-959e053058fb?paneId=startcoding&ace_config=%7B%22region%22%3A%22%22%2C%22orgGuid%22%3A%22cb3f78e5-5fdf-4335-8cd9-2599fdbc153f%22%2C%22spaceGuid%22%3A%22592e6bbf-ea95-4846-ba65-6620e0adcd44%22%2C%22redirect%22%3A%22https%3A%2F%2Fconsole.w3ibm.bluemix.net%2Fdashboard%2Fapps%22%2C%22bluemixUIVersion%22%3A%22Atlas%22%7D)

**Comment by weieigao** [comment]
Hi @lucaskuperman , the above two commands invoke embedded cf cli (6.26.0). 
I cannot reproduce on my side. Could your provide the cf trace (`bx config --trace=true`)?

**Comment by bagrym** [comment]
@lucaskuperman Hi ! have you managed to solved problem 1? I get the same issue.. noticed that bx cf ssh appname -c ls returns root folder structure.. but with bx cf ssh appname cmd is hang forever

**Comment by boyang9527** [comment]
@bagrym  what is version you are using? the `bx cf ssh` hangs on windows will be fixed in coming release.

**Comment by bagrym** [comment]
@boyang9527  I am using IBM Bluemic CLI version 0.6.5 (0183260-2018-02-05T06:56:08+00:00)

**Comment by boyang9527** [comment]
close the issue since the `bx cf ssh` hanging issue has been fixed in v0.6.6
